src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements.mdx

```diff
@@ -0,0 +1,1967 @@
+---
+title: '.NET agent: compatibility and requirements'
+tags:
+  - Agents
+  - NET agent
+  - Getting started
+metaDescription: "APM's .NET agent for .NET Framework applications: compatibility and requirement, and what frameworks are automatically instrumented."
+redirects:
+  - /docs/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-framework
+  - /docs/agents/net-agent/getting-started/compatibility-requirements-net-agent
+  - /docs/agents/net-agent/getting-started/compatibility-requirements-net-framework-agent
+  - /docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-core/#requirements
```

Comment:
Check for this redirect, seems to be broken

Reply:
The link currently exists but it will be broken and redirect here once we get this published out!